rediscluster/client.py

```diff
@@ -850,7 +846,7 @@ class RedisCluster(StrictRedisCluster):
         }
     )
 
-    def pipeline(self, transaction=True, shard_hint=None, use_threads=None):
```

Comment:
This change would require a note in the docs/Upgrading.md file stating that this argument can be removed from the implementers code.